static/src/javascripts/projects/common/modules/discussion/loader.js

```diff
@@ -69,7 +69,7 @@ Loader.prototype.initTopComments = function() {
         this.gotoComment(commentId);
     });
 
-    return fetchJson('/discussion/top-comments/' + this.getDiscussionId() + '.json', {
+    return fetchJson('/discussion/top-comments/' + this.getDiscussionId() + '.json?closed=' + config.page.commentable, {
```

Comment:
There is no whitelist on api.nextgen.guardianapps.co.uk. The whitelist on www.theguardian.com is mainly to deal with campaign codes (which don't apply on this endpoint).